modules/oc-mirror-updating-registry-about.adoc

```diff
@@ -5,34 +5,26 @@
+You can update your mirror registry content by updating the image set configuration file and mirroring the image set to the mirror registry. When you run the oc-mirror plugin again, it generates an image set that only contains new and updated images since the previous execution.
 
-[IMPORTANT]
-====
-Generated image sets are sequential and must be pushed to the target mirror registry in order. You can derive the sequence number from the file name of the generated image set archive file.
-====
+While updating the mirror registry, you must take into account the following considerations:
 
-[discrete]
-== Adding new and updated images
+* Images are pruned from the target mirror registry if they are no longer included in the latest image set that was generated and mirrored. Hence, ensure that you are updating images for the same combination of the following key components so that only a differential image set is created and mirrored:
 
-Depending on the settings in your image set configuration, future executions of oc-mirror can mirror additional new and updated images. Review the settings in your image set configuration to ensure that you are retrieving new versions as necessary. For example, you can set the minimum and maximum versions of Operators to mirror if you want to restrict to specific versions. Alternatively, you can set the minimum version as a starting point to mirror, but keep the version range open so you keep receiving new Operator versions on future executions of oc-mirror. Omitting any minimum or maximum version gives you the full version history of an Operator in a channel. Omitting explicitly named channels gives you all releases in all channels of the specified Operator. Omitting any named Operator gives you the entire catalog of all Operators and all their versions ever released.
+** Image set configuration
 
-All these constraints and conditions are evaluated against the publicly released content by Red Hat on every invocation of oc-mirror. This way, it automatically picks up new releases and entirely new Operators. Constraints can be specified by only listing a desired set of Operators, which will not automatically add other newly released Operators into the mirror set. You can also specify a particular release channel, which limits mirroring to just this channel and not any new channels that have been added. This is important for Operator products, such as Red Hat Quay, that use different release channels for their minor releases. Lastly, you can specify a maximum version of a particular Operator, which causes the tool to only mirror the specified version range so that you do not automatically get any newer releases past the maximum version mirrored. In all these cases, you must update the image set configuration file to broaden the scope of the mirroring of Operators to get other Operators, new channels, and newer versions of Operators to be available in your target registry.
+** Source registry or path
```

Comment:
It's rather the final destination registry here. 
This is either the registry argument of the mirror to mirror command, or the registry argument of the disk to mirror command.

ie. when pruning happens at the end of the mirroring process, it will prune that destination registry.
If the user specifies a different destination registry for the same storage config and imagesetconfig, oc-mirror will attempt to prune images on a registry where it didn't do mirroring before.

Reply:
So it would be `Destination registry`. is that right?